src/presets/nitro-dev.ts

```diff
@@ -8,5 +10,13 @@ export const nitroDev = defineNitroPreset({
   },
   externals: { trace: false },
   inlineDynamicImports: true, // externals plugin limitation
-  sourceMap: true
+  sourceMap: true,
+  hooks: {
+    'nitro:rollup:before' (nitro) {
+      if (isWindows) {
+        // Windows dynamic imports should be file:// url
+        nitro.options.alias['#build'] = pathToFileURL(nitro.options.buildDir).href
```

Comment:
Shared situation with both is `externals: { trace: false }`

Reply:
Hm. Perhaps we could just test for that instead in rollup config, or are you happy with the preset option?

Reply:
Improved in https://github.com/unjs/nitro/commit/1ec2ee4cf3dfd355a2b7d8d6d7aa5de6dcb51b7c. Extending alias was too late here.